### ISSUE TITLE
Successive models generation produces invalid PHP classes.

### ISSUE THREAD

**Issue by tjamps** [phalcon/phalcon-devtools#276]
Hi all,

I'm facing a big problem using Phalcon Dev Tools to generate my models.
When I issue the following command :

```Shell
phalcon model robots --output app/models/ --force
```

The following model gets generated (some lines are omitted for brievety) :
```PHP
<?php
class Robots extends \Phalcon\Mvc\Model
{
    /**
     *
     * @var integer
     */
    public $id;

    // Other properties here...

    /**
     * Initialize method for model.
     */
    public function initialize()
    {
        $this->setSchema("");
    }

    /**
     * Independent Column Mapping.
     */
    public function columnMap()
    {
        return array(
            'id' => 'id', 
            'name' => 'name', 
            'type' => 'type', 
            'year' => 'year'
        );
    }
}
```

So far, so good.
But then, if I issue the exact same command on the same file without changing anything in the database (in other words, the command is issued twice without any other operation taking place in between), the following code gets generated :

```PHP
<?php

class Robots extends \Phalcon\Mvc\Model
{

    /**
     *
     * @var integer
     */
    public $id;

    // Other properties here...
    public function initialize()
    {
        $this->setSchema("");
    }
    public function columnMap()
    {
        return array(
            'id' => 'id', 
            'name' => 'name', 
            'type' => 'type', 
            'year' => 'year'
        );
    }

    /**
     * Independent Column Mapping.
     */
    public function columnMap()
    {
        return array(
            'id' => 'id', 
            'name' => 'name', 
            'type' => 'type', 
            'year' => 'year'
        );
    }
}
```
Note that the function `columnMap` is added twice to the model, which of course is not allowed by PHP.

Any idea on what I'm doing wrong ?

Test environnement
OS: Ubuntu 14.04.1 LTS
PHP: 5.5.9
PostgreSQL : 9.3
Phalcon : 1.3.4
Phalcon Dev Tools : commit 070c3c3a51c81944ae241267e271ee9ce8ebf066

**Comment by 7thcubic** [comment]
weird. i submitted a pull for this, doesnt show up though

**Comment by rianorie** [comment]
#296 :)

**Comment by sergeyklay** [comment]
Fixed #392
